versioned_docs/version-8.3/reference/supported-environments.md

```diff
@@ -33,49 +39,86 @@ For example, 1.2+ means support for the minor version 2, and any higher minors (
 
 ## Camunda 8 Self-Managed
 
-We highly recommend running Camunda 8 Self-Managed in a Kubernetes environment. We provide officially supported [Helm Charts](/self-managed/platform-deployment/helm-kubernetes/overview.md) for this. Please follow the [Installation Guide](/self-managed/platform-deployment/overview.md) to learn more about installation possibilities.
+### Deployment options
+
+- [Helm/Kubernetes](/self-managed/platform-deployment/helm-kubernetes/overview.md) [recommended]
```

Comment:
Link to K8S certification link [https://www.cncf.io/training/certification/software-conformance/](https://www.google.com/url?q=https://www.cncf.io/training/certification/software-conformance/&sa=D&source=docs&ust=1708512503126547&usg=AOvVaw2RLHpL0yFSylGOH-mfdFYj)

Reply:
I linked to the subsection on benefits since I thought that was most relevant for our customers who weren't familiar with Certified Kubernetes - https://www.cncf.io/training/certification/software-conformance/#benefits